concent_api/concent_api/settings/base.py

```diff
@@ -360,3 +360,6 @@
 
 # Define if exception should be raised when database transactions are nested.
 DETECT_NESTED_TRANSACTIONS = True
+
+# Define how long concent wait for middleman response before raising exception.
+SCI_CALLBACK_RETRIES_TIME = 60
```

Comment:
Should be `SCI_CALLBACK_RETRY_TIME`, because this is the time of single retry, not many.

Reply:
Actually it is time of many retries. It's going to be hard to explain without looking at the code. The way @dybi explain it to me, is that we already have a timeout in `core/constants.py` in form of:
```
# Defines how many seconds should SCI callback wait for response from MiddleMan.
SCI_CALLBACK_MAXIMUM_TIMEOUT = 30
```
And now, depending on how long `SCI_CALLBACK_RETRIES_TIME` will be, that how many times retry will happen, or in other words how many times  `SCI_CALLBACK_MAXIMUM_TIMEOUT` will fit inside `SCI_CALLBACK_RETRIES_TIME`.

Reply:
Right, I have checked implementation but forgot to go back to this comment, thanks for explanation.